### PR TITLE
Added biography

### PR DESCRIPTION
**Pull request by mtlynch4**
Writing the biography in Markdown was fairly easy. A guide to Markdown syntax that I found here (https://www.markdownguide.org/basic-syntax/) was very helpful. 

Initially, I mistakenly cloned the original Repository instead of forking first, which I realized upon attempting to push and was told I was attempting to push not to my account, and corrected the mistake.

Then, I forked first (from the recreated repo), cloned the repository locally, and copied what I had written earlier into the new Read.me, since it was still saved locally, just not pushed to Git. 

Overall, the file management aspect of this assignment was more challenging than the syntax of Markdown, and likely more helpful as well. I feel much more comfortable with my workflow now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
\ No newline at end of file
+# hw01
+<h2> Michael Lynch </h2>
+<h3>Name</h3>
+I like my name, but it is absurdly common. Not only am I the fourth Michael Lynch in a row (although not Michael the Fourth, since my middle name is different), I am one of nearly *5 million people* in the United States named Michael. That's more people than the **entire state** of Alabama.
+
+<h3>Geography</h3>
+I was born in Boston, Massachusetts, and lived there until I was four, when my family moved to Georgia so my dad could work for the Centers for Disease Control. I grew up in Dunwoody, Georgia, a suburb north of Atlanta. There's not much there other than some trees and hills, but Ryan Seacrest went to my high school. So that's fun. 
+
+![These two towers are called the King and Queen](C:\Users\mtlyn\Pictures\Book Cover\Downtown dunwoody.jpg "Downtown Dunwoody")
```
Comment (yanwenwang24) [inline README.md:9]: I think you should use the path of Github here, instead of the path of your drive.

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+<h3>School</h3>
+I'm doing a Joint BA/MA here at UChicago, in Economics and International Relations. This class counts towards the latter. 
+
+<h3>Interests</h3>
+In my free time, I enjoy reading, mostly the news, which makes me sad, and fantasy, which doesn't (usually). 
+
+<h4>Favorite authors from this summer</h4>
+1. Ann Leckie
```
Comment (yanwenwang24) [inline README.md:18]: The list here is not displayed as expected.

**Review by yanwenwang24 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     *       |            |
| **Coding strategy**         |           |      *        |            |
| **Presentation**            |    *       |              |            |
| **Achievement, creativity** |    *       |              |            |
| **Ease of access**          |    *       |              |            |

The link failed to work. You need to make sure the image is in the project folder and set the path accordingly. 
The list is not displayed properly.

Check

File: README.md [hunk 3]
```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+<h2> Michael Lynch </h2>
+<h3>Name</h3>
```
Comment (zxgneil) [inline README.md:3]: Markdown language in itself is sufficient in rendering headings. I suspect that adding HTML language along with the markdown language is what prevented your changes from showing.

File: README.md (hunk 1 above)
Comment (zxgneil) [reply README.md:9]: Yes. You are using an absolute file path that makes github failing at locating your image. Try using a relative file path and uploading the image next time.

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |              |      x      |
| **Achievement, creativity** |           |         x     |            |
| **Ease of access**          |           |       x       |            |

Remarks:
1. None of the change you made is successfully rendering, in part because you have a mixed usage of markdown and html language. Please stick with markdown for now. 
1. You used an absolute path so that the image would not render even if you only use markdown language. Try use a relative file path next time!
1. Try committing more in future assignments because that help you (and others) keeping track of your work progress.

**Pull request closed without merging**